CustomDropDown/Model/DropDownConfig.swift

```diff
@@ -14,6 +14,11 @@ public struct DropDownConfig {
     public var selectedLabelTag: Int = 9999
     public var dropDownLeftRightPadding: UIEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 0, right: 10)
     public var dropDownMode: DropDownMode = .label
+    public var dropDownShadowColor: CGColor = UIColor.black.cgColor
```

Comment:
Can we have all the config of these shadow color, offset and corner radius in a new model?

Like create a struct named **CORNERANDSHADOWCONFIG** give it a good name as you wish, but put the property name in config as '_shadowAndCornerConfig: structType_'

Reply:
Sure! it really got better organized with the new model.